dfc/proxy.go

```diff
@@ -608,7 +598,23 @@ func (p *proxyrunner) httpbckdelete(w http.ResponseWriter, r *http.Request) {
 	}
 	switch msg.Action {
 	case ActDestroyLB:
-		p.deleteLocalBucket(w, r, bucket)
+		if !p.islocalBucket(bucket) {
+			p.invalmsghdlr(w, r, fmt.Sprintf("Cannot delete non-local bucket %s", bucket))
+			return
+		}
+		lbmapLock.Lock()
+		defer lbmapLock.Unlock()
+		if !p.lbmap.del(bucket) {
+			s := fmt.Sprintf("Local bucket %s does not exist", bucket)
+			p.invalmsghdlr(w, r, s)
+			return
+		}
+		if errstr := p.savelbmapconf(); errstr != "" {
+			p.lbmap.add(bucket)
+			p.invalmsghdlr(w, r, errstr)
```

Comment:
return missing?
The similar code below for create LB does 'return' in this case.